nvm.sh

```diff
@@ -704,6 +708,91 @@ nvm_binary_available() {
   nvm_version_greater_than_or_equal_to "$(nvm_strip_iojs_prefix "${1-}")" v0.8.6
 }
 
+nvm_set_colors() {
+  if nvm_has_colors; then
+    local COLORS
+    local INSTALLED_COLOR
+    local LTS_AND_SYSTEM_COLOR
+    local CURRENT_COLOR
+    local NOT_INSTALLED_COLOR
+    local DEFAULT_COLOR
+
+    COLORS=$1
+    INSTALLED_COLOR="${COLORS:0:1}"
+    LTS_AND_SYSTEM_COLOR="${COLORS:1:1}"
+    CURRENT_COLOR="${COLORS:2:1}"
+    NOT_INSTALLED_COLOR="${COLORS:3:1}"
+    DEFAULT_COLOR="${COLORS:4:1}"
+
+    CONFIRMATION_MESSAGE="Setting colors to: \033[$(nvm_print_color_code "${INSTALLED_COLOR}") ${INSTALLED_COLOR}\033[$(nvm_print_color_code "${LTS_AND_SYSTEM_COLOR}") ${LTS_AND_SYSTEM_COLOR}\033[$(nvm_print_color_code "${CURRENT_COLOR}") ${CURRENT_COLOR}\033[$(nvm_print_color_code "${NOT_INSTALLED_COLOR}") ${NOT_INSTALLED_COLOR}\033[$(nvm_print_color_code "${DEFAULT_COLOR}") ${DEFAULT_COLOR}\033[0m"
+    nvm_echo_with_colors "$CONFIRMATION_MESSAGE"
+    export NVM_COLORS=$1
+  fi
+}
+
+nvm_get_colors() {
+  if [ -n "${NVM_COLORS-}" ]; then
+    NVM_INSTALLED_COLOR=$(nvm_print_color_code "${NVM_COLORS:0:1}")
+    NVM_SYSTEM_COLOR=$(nvm_print_color_code "${NVM_COLORS:1:1}")
+    NVM_CURRENT_COLOR=$(nvm_print_color_code "${NVM_COLORS:2:1}")
+    NVM_NOT_INSTALLED_COLOR=$(nvm_print_color_code "${NVM_COLORS:3:1}")
+    NVM_DEFAULT_COLOR=$(nvm_print_color_code "${NVM_COLORS:4:1}")
+
+    NVM_LTS_COLOR=$(echo "$NVM_SYSTEM_COLOR" | tr '0;' '1;')
+  else
+    NVM_CURRENT_COLOR="0;32m"
+    NVM_INSTALLED_COLOR="0;34m"
+    NVM_LTS_COLOR="1;33m"
+    NVM_NOT_INSTALLED_COLOR="0;31m"
+    NVM_DEFAULT_COLOR="0;37m"
+    NVM_SYSTEM_COLOR="0;33m"
+  fi
+}
+
+nvm_print_color_code() {
+  case ${1} in
+    r) echo "0;31m";;
+    R) echo "1;31m";;
+    g) echo "0;32m";;
+    G) echo "1;32m";;
+    b) echo "0;34m";;
+    B) echo "1;34m";;
+    c) echo "0;36m";;
+    C) echo "1;36m";;
+    m) echo "0;35m";;
+    M) echo "1;35m";;
+    y) echo "0;33m";;
+    Y) echo "1;33m";;
+    k) echo "0;30m";;
+    K) echo "1;30m";;
+    e) echo "0;37m";;
+    W) echo "1;37m";;
+    *) echo "Invalid color code"; return 1;;
```

Comment:
```suggestion
    r) nvm_echo "0;31m";;
    R) nvm_echo "1;31m";;
    g) nvm_echo "0;32m";;
    G) nvm_echo "1;32m";;
    b) nvm_echo "0;34m";;
    B) nvm_echo "1;34m";;
    c) nvm_echo "0;36m";;
    C) nvm_echo "1;36m";;
    m) nvm_echo "0;35m";;
    M) nvm_echo "1;35m";;
    y) nvm_echo "0;33m";;
    Y) nvm_echo "1;33m";;
    k) nvm_echo "0;30m";;
    K) nvm_echo "1;30m";;
    e) nvm_echo "0;37m";;
    W) nvm_echo "1;37m";;
    *) nvm_echo "Invalid color code"; return 1;;
```

Reply:
Changed to `nvm_echo` and single-quoted strings.